docker/build_container_release.sh

```diff
@@ -24,10 +24,14 @@ export DOCKER_IMAGE_NAME=${DOCKER_IMAGE_NAME:-"nvcr.io/nvidia/morpheus/morpheus"
 export DOCKER_IMAGE_TAG=${DOCKER_IMAGE_TAG:-"$(git describe --tags --abbrev=0)-runtime"}
 export DOCKER_TARGET=${DOCKER_TARGET:-"runtime"}
 
-popd &> /dev/null
+export GIT_URL=${GIT_URL:-$(git remote get-url origin)}
```

Comment:
This seems problematic based on the name of the remote and would be improved with a local clone